### PR TITLE
Add Dockerfile and some readme instructions

### PR DESCRIPTION
**Pull request by jaywink**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 .idea/
+mon.yaml
+_Deparsed_XSubs.pm
```
Comment (michaelkaye) [inline .gitignore:3]: Do you not also want to exclude this from the docker container?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:18.04
```
Comment (michaelkaye) [inline Dockerfile]: Would there be value in using FROM perl:5   as a base image? Save on the apt installing

**Review by michaelkaye — COMMENTED** [review]
I mean, it seems OK, few little things, and not sure if there's a reason not to use the base perl images rather than start from an ubuntu.

File: Dockerfile (hunk 2 above)
Comment (michaelkaye) [reply Dockerfile]: note, i don't really worry overmuch about this, if anything it'll reduce image size and remove needing to set various envvars, but ...

Comment (jaywink) [reply Dockerfile]: I did start with that but had problems getting it to work. Might have been a combination of SSL libraries and the env variables. The env variables are still (at least some of them) needed for things to work. Updated with perl:5 now :+1:

**Review by michaelkaye — APPROVED** [review]
compiles, runs (bails out because domain.tld isn't a valid domain)

**Pull request merged**